src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -170,7 +170,9 @@ export default class DefaultAudioVideoController
       this,
       this._logger
     );
-    this._audioMixController = new DefaultAudioMixController(this._logger);
+    const audioMixController = new DefaultAudioMixController(this._logger);
+    this._audioMixController = audioMixController;
+    this._mediaStreamBroker.addMediaStreamBrokerObserver(audioMixController);
```

Comment:
Should we also remove the observer in the `destroy` method in the case the customer calls `meetingSession.destory`?

https://github.com/aws/amazon-chime-sdk-js/blob/ae910a680c8d0de920c8bdc639e43f37eb621446/src/meetingsession/DefaultMeetingSession.ts#L135-L137

Reply:
Oops. Forget about it. Updated to remove during destroy.